### DISCUSSION TITLE
Multiple park entrance types

### DISCUSSION THREAD

**Discussion by Broxzier** [OpenRCT2/OpenRCT2#15750] [Feature requests] [answered]
### Describe the feature

Currently the game supports only one entrance type per map. With the new save format, multiple park entrance types should be able to be selected.

Choosing the style to use could be either implemented in the map window in the scenario editor (set once placed), or be made customizable in the park window, much like how you'd set a ride's entrance style (editable after placing). The later could be exposed to the players during normal gameplay, to let them change the entrance's appearance.

### Feature type

Quality of life

**Comment by karst** [comment]
I'd like to bump thisone. I don't know how hard it is to implement, but there's cases where there's different park entrances depending on what themed area you enter the park. Phantasialand is the perfect example.

**Comment by Gymnasiast** [comment, marked as answer]
https://github.com/OpenRCT2/OpenRCT2/pull/21909
